shell/config/router.js

```diff
@@ -99,7 +99,7 @@ export const routerOptions = {
           name:      'c-cluster-ecm'
         },
         {
-          path:      'settings',
```

Comment:
This configuration wasn't actually being used correctly. It was falling back on /c/:cluster/:product

![settings](https://github.com/rancher/dashboard/assets/55104481/5f6ec7d8-bfe8-4333-ba51-c230500bfc0a)